sdks/python/apache_beam/ml/inference/sklearn_inference_test.py

```diff
@@ -47,6 +47,12 @@
 from apache_beam.testing.util import assert_that
 from apache_beam.testing.util import equal_to
 
+# pylint: disable=wrong-import-order, wrong-import-position, ungrouped-imports, unused-import
+try:
+  from apache_beam.io.gcp.gcsfilesystem import GCSFileSystem
```

Comment:
why is this necessary? I am not seeing any usage of GCSFileSystem

Reply:
This might have been preemptive. Unless we are able to save files, I think these tests could fail.  However, it doesn't seem relevant to this PR and we should defer fixing it until it becomes an issue.